code/dynamic_programming/src/edit_distance/edit_distance.js

```diff
@@ -0,0 +1,45 @@
+// dynamic programming | edit distance | Javascript
+// Part of Cosmos by OpenGenus Foundation
+
+editDistance = (str1, str2) => {
```

Comment:
I actually would _not_ recommend using an arrow function here, because those are usually used for small one-liners (and anonymous functions).

```suggestion
function editDistance(str1, str2) {
```